styles.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
+}
+
+body {
+  background-color: #ff66c4;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  width: 600px;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+h2,
+h4 {
+  font-weight: 600;
+  font-size: 36px;
+  margin-bottom: 50px;
+}
+
+h4 {
+  font-size: 26px;
+  margin-bottom: 20px;
+}
+
+.inputButtonContainer {
+  display: grid;
+  grid-template-columns: 1fr 150px;
+  gap: 20px;
+  width: 85%;
+  margin-bottom: 50px;
+}
+
+input {
```

Comment:
nit: as much as is possible, try to style using classes and avoid styling elements directly.